src/print.js

```diff
@@ -0,0 +1,3 @@
+export default function printMe() {
+  console.log('I get called from print.js!');
+}
```

Comment:
- [ ] Kindly try to delete the console.log statements. It is not a good practice to push it with code.